.github/workflows/ci.yaml

```diff
@@ -0,0 +1,84 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: ci
+jobs:
+  units:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [7, 8, 11]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - run: java -version
+    - run: .kokoro/build.sh
+      env:
+        JOB_TYPE: test
```

Comment:
Let's remove the coverage build below and add the codecov action like: https://github.com/googleapis/nodejs-asset/blob/master/.github/workflows/ci.yaml#L21-L25

Reply:
done